docs/portal/team-members.md

```diff
@@ -20,15 +20,25 @@ In the UID2 Portal, on the **Team Members** page, Admin users can perform all ac
 
 To access the Team Members page, go to the top right of the UID2 Portal user interface where your name is displayed. Click the down arrow and choose **Manage Team Members**.
 
+For each team member, the following information is displayed:
+- Name
+- Email
+- Job Function
+- Role
+- Actions: the available actions are **Edit** or **Delete**.
```

Comment:
Resend invite is also under the actions tab

Reply:
Hi @ashleysmithTTD: Thx! Slippery one. Only available for pending users. Updated.